concepts/cookbook/ffp_transfusion.sql

```diff
@@ -0,0 +1,69 @@
+-- --------------------------------------------------------
+-- Title: Retrieves instances of FFP transfusions
+-- Notes: this query does not specify a schema. To run it on your local
+-- MIMIC schema, run the following command:
+--  SET SEARCH_PATH TO mimiciii;
+-- Where "mimiciii" is the name of your schema, and may be different.
+-- --------------------------------------------------------
+
+
+drop materialized view if exists ffp_transfusion cascade; 
+create materialized view ffp_transfusion as
+
+with raw_ffp as (
+  select amount
+    , amountuom
+    , icustay_id
+    , charttime as tsp
+  from inputevents_cv
+  where itemid in (
+      30005,  -- Fresh Frozen Plasma
+      30180,  -- Fresh Froz Plasma
+      44172,  -- FFP GTT         
+      44236,  -- E.R. FFP        
+      46410,  -- angio FFP
+      46418,  -- ER ffp
+      46684,  -- ER FFP
+      44819,  -- FFP ON FARR 2
+      46530,  -- Floor FFP       
```

Comment:
this line and the 6 above it all seem to be emergency room or floor transfusions